app/util/logs/index.ts

```diff
@@ -25,24 +25,24 @@ export const generateStateLogs = (state: any, loggedIn = true): string => {
   delete fullState.engine.backgroundState.PhishingController;
   delete fullState.engine.backgroundState.AssetsContractController;
 
-  // Remove encrypted vault from logs
-  delete fullState.engine.backgroundState.KeyringController.vault;
+  // Remove Keyring controller data
```

Comment:
Nit: The previous comment did a better job of explaining _why_ we're removing state. Comments that explain what we're doing are less helpful (it's clear already from reading the code that we're removing keyring controller state).

Here's a suggested improvement:
```suggestion
  // Remove Keyring controller data so that encrypted vault is not included in logs
```

Reply:
Love it! Thanks!